### PR TITLE
chore(markdown-lint): enable the `no-duplicate-heading` rule

### PR DESCRIPTION
**Pull request by yin1999**
### Description

enable the `no-duplicate-heading` rule. The rule has been enabled in [upstream](https://github.com/mdn/content/blob/1f39f655c624e433c4bb78bee587c5ff99398688/.markdownlint.jsonc#L19-L21).

### Related issues and pull requests

Preparation:

- es: #17617
- fr: #17581
- ja: #17627
- ko: #17716
- pt-br: #17667
- ru: #17699
- zh-cn: #17560
- zh-tw: #17561

### CODE REVIEW

File: files/ja/games/index.md [hunk 1]
```diff
@@ -51,7 +51,5 @@ MDN ゲーム開発センターにようこそ! サイトのこのエリアで
   - : ファイルサイズが 13 KB 以内であること、という制約のもと開発された HTML ゲームを対象とした JavaScript の競技プログラミングサイトです。登録されているゲームは全て、GitHub に公開されています。
 - [Mozilla Hacks Blog](https://hacks.mozilla.org/category/games/)
   - : Mozilla Hacks ブログの Games カテゴリには、ゲーム開発に関する興味深い記事が掲載されています。
-
-## 関連情報
-
 - [Games section on wiki.mozilla.org](https://wiki.mozilla.org/Platform/Games)
+  - : Mozilla のプラットフォーム ゲームへの関与に関する情報を含む Wiki ページ。
```
Comment (yin1999) [inline files/ja/games/index.md:55]: Please check the translation as I'm not a native speaker (just use Google Translate).

**Review by potappo — APPROVED** [review]
I updated the translation  in order not to misunderstand "platform".

**Comment by yin1999** [comment]
Thank you @potappo :)

**Comment by yin1999** [comment]
@queengooborg could you have a look on this :)

**Review by queengooborg — APPROVED** [review]
Thank you, LGTM!

**Pull request merged**